test/framework/kogitoapp.go

```diff
@@ -30,61 +31,125 @@ const (
 	mavenMirrorURLEnvVar  = "MAVEN_MIRROR_URL"
 )
 
-// DeployQuarkusExample deploy a Quarkus example
-func DeployQuarkusExample(namespace, appName, contextDir string, native, persistence, events bool, labels map[string]string) error {
-	GetLogger(namespace).Infof("Deploy quarkus example %s with name %s, native %v and persistence %v", contextDir, appName, native, persistence)
-	return DeployExample(namespace, appName, contextDir, "quarkus", native, persistence, events, labels)
+// KogitoAppDeploymentRuntime defines the runtime for a KogitoAppDeployment
+type KogitoAppDeploymentRuntime string
+
+const (
+	// QuarkusKogitoAppDeploymentRuntime is quarkus deployment runtime
+	QuarkusKogitoAppDeploymentRuntime = "quarkus"
```

Comment:
We have this const in the `api` package, in the `KogitoApp` CR.

Reply:
I was asking myself about that. My thinking at that moment is that I did not want to depend on that. but thinking again it does not make sense ^^